packages/dev/parcel-transformer-s2-icon/IconTransformer.js

```diff
@@ -66,7 +66,7 @@ function template(asset, iconName, svg) {
   }
   return (
 `
-import {IconProps, IconContext, IconContextValue} from '~/src/Icon';
+import {IconProps, IconContext, IconContextValue} from '${asset.filePath.includes('@react-spectrum/s2') ? '~/src/Icon' : '@react-spectrum/s2'}';
```

Comment:
So I ran the tool locally and outputted the results to a test folder at the root of the project. Is this `~/src/Icon` supposed to point to the where our `s2` lives in the repo (aka packages/@react-spectrum/s2/)? I'm assuming this import needs to resolve to `Icon.tsx` essentially

Additionally, I noticed that it imported SVGProps twice

```
import {SVGProps, useRef} from 'react';
import {useContextProps} from 'react-aria-components';

import { SVGProps } from "react";
```

Reply:
`~/src/Icon` is only for local development for us
for all other users of this, it'll point to `@react-spectrum/s2`

Reply:
not sure why that import changed from when we were in our other repo, but I've updated it to account for this line